cmd/soroban-cli/src/utils.rs

```diff
@@ -29,6 +32,44 @@ pub fn transaction_hash(tx: &Transaction, network_passphrase: &str) -> Result<[u
     Ok(Sha256::digest(signature_payload.to_xdr(Limits::none())?).into())
 }
 
+static EXPLORERS: phf::Map<&'static str, &'static str> = phf_map! {
+    "Test SDF Network ; September 2015" => "https://stellar.expert/explorer/testnet",
+    "Public Global Stellar Network ; September 2015" => "https://stellar.expert/explorer/public",
+};
+
+pub fn explorer_url_for_transaction(network: &Network, tx_hash: &str) -> Option<String> {
+    EXPLORERS
+        .get(&network.network_passphrase)
+        .map(|base_url| format!("{base_url}/tx/{tx_hash}"))
+}
+
+pub fn explorer_url_for_contract(network: &Network, contract_id: &str) -> Option<String> {
+    EXPLORERS
+        .get(&network.network_passphrase)
+        .map(|base_url| format!("{base_url}/contract/{contract_id}"))
+}
+
+/// # Errors
+///
+/// Might return an error
+pub fn log_transaction(
+    output: &Output,
+    tx: &Transaction,
+    network: &Network,
+    show_link: bool,
```

Comment:
It seems that this function does nothing if this boolean is false. Do we need to call it all in the case where it is false?

Reply:
Fixed! It should output the transaction hash. 😬